medusa/subtitle_providers/legendastv.py

```diff
@@ -52,6 +52,12 @@
 releases_key = __name__ + ':releases|{archive_id}|{archive_name}'
 
 
+#: Check if the value should actually be cached in dogpile or not
+def should_cache_titles(titles):
```

Comment:
In the end this could be a simple lambda :

```
@region.cache_on_arguments(expiration_time=SHOW_EXPIRATION_TIME, should_cache_fn=lambda value: value)
```

That makes even simpler, right?